app/common/data/models/__init__.py

```diff
@@ -74,4 +76,49 @@ class CollectionSchema(BaseModel):
     created_by_id: Mapped[uuid.UUID] = mapped_column(ForeignKey("user.id"))
     created_by: Mapped[User] = relationship("User")
 
+    sections: Mapped[list["Section"]] = relationship(
+        "Section", lazy=True, order_by="Section.order", collection_class=ordering_list("order", count_from=1)
+    )
+
+    @property
+    def slug(self) -> str:
+        return slugify(self.name)
```

Comment:
This isn't used anywhere yet - is this pre-emptive or about to be used? I have a few concerns - if two sections slugify to the same thing, how will we disambiguate? I think if/when we want to have a slug, it needs to go in the DB with a unique constraint so that we can close off any concerns about dynamically-generated slugs overlapping.

Reply:
Yeah I think this was a placeholder when I was trying stuff out, will remove it